content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,468 @@
+
+.. seealso::
+   :doc:`PEC email <italy_IT>`
+
+e-Invoicing
+-----------
 
-Setup the Codice Destinatario on FatturaPA
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The :abbr:`SdI (Sistema di Interscambio)` is the electronic invoicing system used in Italy. It
+enables to send and receive electronic invoices to and from customers. The documents must be in XML
+format and formally validated by the system before being delivered.
 
-To receive invoices and notifications from third parties, you need to inform the FatturaPA service
-that Odoo is the allowed party to process files for you. To do so, you must setup Odoo's *Codice
-Destinatario* on the FatturaPA portal. The *Codice Destinatario* is ``K95IV18``.
+To be able to receive invoices and notifications, the :abbr:`SdI (Sistema di Interscambio)` service
+must be notified that the user's files are to be sent to **Odoo**, and are processed on their
+behalf. To do so, you must set up Odoo's :guilabel:`Codice Destinatario` on the **Agenzia Delle
+Entrate** portal.
 
-#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate.
-#. Go to section :menuselection:`Fatture e Corrispettivi`.
-#. Set the user as Legal Party for the VAT number you wish to configure the electronic adress.
+#. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate;
+#. Go to section :menuselection:`Fatture e Corrispettivi`;
+#. Set the user as Legal Party for the VAT number you wish to configure the electronic address;
 #. In :menuselection:`Servizi Disponibili --> Fatturazione Elettronica --> Registrazione
-   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, input Odoo's Codice
-   Destinatario (**K95IV18**), then confirm.
+   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, insert Odoo's
+   :guilabel:`Codice Destinatario` `K95IV18`, and confirm.
 
-Give Odoo permission to process files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Electronic Data Interchange (EDI)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
-your database, you need to authorize Odoo to process your files from your database.
+Odoo uses the **FatturaPA** :abbr:`EDI (Electronic Data Interchange)` format for the Italian
+localization, and is enabled by default when installed. When the **file processing authorization**
+has been set, all **invoices** and **bills** are automatically sent for journals with
+:guilabel:`Electronic invoicing` enabled (:menuselection:`Accounting --> Configuration --> Journals
+--> (Select a journal) --> Advanced Settings --> Electronic Data Interchange)`.
 
-To do this, go to :menuselection:`Accounting --> Settings --> Electronic Invoicing` and click on
-*Register*.
+You can check the current status of an invoice by the :guilabel:`Electronic invoicing` field *(cf.
+screenshot below)*. The XML file can be found in the **chatter** of the invoice.
```

Comment:
Please highlight the `Electronic invoicing` field in the image, and correct that the XML can be found in the attachments. (I can't see it in the chatter)
You can use my example or fiddle with this runbot link from which I've taken the example.

![image](https://user-images.githubusercontent.com/1665365/215095819-2a7187f7-f0a0-4933-b92e-f57df824253a.png)

https://23239408-16-0-all.runbot178.odoo.com/web#id=186&cids=6&menu_id=632&action=557&model=account.move&view_type=form